docs-new/docs/graphql/core/migrations/config-v2/reference/metadata-format.mdx

```diff
@@ -0,0 +1,311 @@
+---
+sidebar_label: Metadata format
+sidebar_position: 3
+description: Hasura Metadata file format reference
+keywords:
+  - hasura
+  - docs
+  - metadata
+  - file format
+---
+
+# Metadata format reference (config v2)
+
+## Introduction
+
+With `config v2`, the metadata that is exported from the server by the
+CLI is a directory of multiple files.
+
+:::info Note
+
+For `config v1`, see [Metadata file format reference (config v1)](/graphql/core/migrations/config-v1/reference/metadata-file-format.mdx).
+
+:::
+
+## Metadata directory format
+
+The following files will be generated in the `metadata/` directory of
+your project:
+
+- [actions.graphql](#actionsgraphql)
+- [actions.yaml](#actionsyaml)
+- [allow_list.yaml](#allow_listyaml)
+- [cron_triggers.yaml](#cron_triggersyaml)
+- [functions.yaml](#functions-yaml)
+- [query_collections.yaml](#query_collectionsyaml)
+- [remote_schemas.yaml](#remote_schemasyaml)
+- [tables.yaml](#tables-yaml)
+- [version.yaml](#versionyaml)
+
+:::info Note
+
+The output of the [export_metadata](/graphql/core/api-reference/metadata-api/manage-metadata.mdx) metadata API is a JSON
```

Comment:
missing hash-param - this links to a sun section in the linked doc, pls add hash param for correct section (`#metadata-export-metadata`)